<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>601</width>
     <height>451</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>Sign Up</string>
   </property>
   <widget class="QComboBox" name="comboBox_organization">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Select Organization</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HSM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MITSCA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>JPCS-MIT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IECEP-MIT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MICRO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EIR-AS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PICE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ACI</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IEEE-MIT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PSME</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PICHE-JCMIT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CHE-CHM Thinkers</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IIEE-MIT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PROMAP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>UAPSA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CCC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ORSP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MIT-MC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EECE-Thinkers</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MEMO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MatES-MIT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>YFC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ENC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PIIE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ICPMA-MSC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PSM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EASTS-TSSP MSC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ASHRAE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AGGEM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>G-CLUB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PSSE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SAE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CBI</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CYA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CEGE Thinkers</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ASME</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CHEMSOC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MNS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>J-ZONE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BIOLOGIC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MPS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SME</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MAD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mapua-ICpEP-SE</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit_emailAddress">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Email Address</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_givenName">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Given Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_middleName">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Middle Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_lastName">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_contactNum">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Contact Number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_studentNumber">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Student Number</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_signUp">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>390</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string> Sign Up</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>521</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Account Type</string>
    </property>
    <widget class="QRadioButton" name="radioButton_User">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Student/Professor</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_admin">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Adminstrator</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit_userName">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_lastName</tabstop>
  <tabstop>lineEdit_givenName</tabstop>
  <tabstop>lineEdit_middleName</tabstop>
  <tabstop>lineEdit_userName</tabstop>
  <tabstop>lineEdit_emailAddress</tabstop>
  <tabstop>lineEdit_password</tabstop>
  <tabstop>lineEdit_studentNumber</tabstop>
  <tabstop>lineEdit_contactNum</tabstop>
  <tabstop>comboBox_organization</tabstop>
  <tabstop>radioButton_User</tabstop>
  <tabstop>radioButton_admin</tabstop>
  <tabstop>pushButton_signUp</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
